exercises/java/encryption-context-complete/.gitignore

```diff
@@ -0,0 +1,3 @@
+/bin/
```

Comment:
why `/bin/` and not `bin/` ? I don't think there is a difference other than the inconsistency.

Reply:
I think this was generated by Maven. I don't remember writing it. Anyway I will drop the trailing slash.